src/ILCompiler.Compiler/src/Compiler/DependencyAnalysis/GCStaticsNode.cs

```diff
@@ -73,11 +84,22 @@ public override ObjectData GetData(NodeFactory factory, bool relocsOnly = false)
 
             if (factory.Target.Abi == TargetAbi.CoreRT)
             {
-                builder.EmitPointerReloc(GetGCStaticEETypeNode(factory), 1);
+                int delta = 1;
```

Comment:
> Didn't realize we could use the runtime constants in the compiler. Looks like we just include it. that's pretty neat.

Yup, I started pushing for it because it makes it way easier to connect the places in the classlib that use something with the place in the compiler that emits it.